eventdata/parameter_sources/elasticlogs_kibana_source.py

```diff
@@ -150,7 +155,7 @@ def __init__(self, track, params, **kwargs):
             self._window_end = [{"type": "relative", "offset_ms": 0}]
 
     def partition(self, partition_index, total_partitions):
-        seed = partition_index * self._params["seed"] if "seed" in self._params else None
```

Comment:
When does it called? It seems that in my local test it. gets called with the **same** partition_index a few times.

Reply:
This method is called before a task is executed by a client and it is called once per task and simulated client. See https://github.com/elastic/rally/blob/859af867f0f8863402c2199c09044951a0417211/esrally/driver/driver.py#L1664

Reply:
thank you